### DISCUSSION TITLE
为啥window本地启动debug，提示

### DISCUSSION THREAD

**Discussion by wuyu8995861** [apache/streampark#3140] [Q&A]
FLINK_PLUGIS_DIR  IS NOT  Empty

**Comment by caicancai** [comment]
Hello, can you post a picture of the error report?

**Comment by wuyu8995861** [comment]
![1695144493916](https://github.com/apache/incubator-streampark/assets/26186544/9b73a300-a43f-4ffa-a544-d0db9d03f72d)
![1695144667846](https://github.com/apache/incubator-streampark/assets/26186544/680e51bd-2315-4554-971d-9f060d45240d)
  my is wsl2 deploy flink-1.13.6

**Comment by wuyu8995861** [comment]
check dist_jar ?

**Comment by wolfboys** [comment]
> FLINK_PLUGIS_DIR IS NOT Empty

There is too little key information. Please submit an issue and provide more details about the environment info (flink version, os, streampark version, etc...) and error logs.
